lib/egg.js

```diff
@@ -317,13 +322,13 @@ class EggCore extends KoaApplication {
    * @param  {Function} fn The inputted function.
    * @return {AsyncFunction} An async promise-based function.
    * @example
-   * ```javascript
-   *  const fn = function* (arg) {
+    ```javascript
```

Comment:
```js

Reply:
I'll fix it later, because I found there's something pending before, I'll try to fix and submit again.

Reply:
In the scope of javascript, we don't need '*'.

Reply:
or we can just remove this example code because I think it is needless here, just add a comment like `transform generator function to async function` is enough.